app/models/participant_profile/ecf.rb

```diff
@@ -203,8 +203,8 @@ def sync_status_with_induction_record
     induction_record&.update!(mentor_profile:) if saved_change_to_mentor_profile_id?
   end
 
-  def update_declaration_temp_types!
-    participant_declarations.update_all(temp_type: type.sub("ParticipantProfile", "ParticipantDeclaration"))
+  def update_declaration_types!
+    participant_declarations.update_all(type: type.sub("ParticipantProfile", "ParticipantDeclaration"))
```

Comment:
I think this should be temp type 🤔

Reply:
or actually we should be changing both

Reply:
yeah this is a mistake it should be `temp_type`, I'll update. We don't need to update `type` as we're going to backfill over it anyway in the next PR